testutil/docker.go

```diff
@@ -270,3 +270,24 @@ func DockerInspect(containerID string) (types.ContainerJSON, error) {
 	x.Check(err)
 	return cli.ContainerInspect(context.Background(), containerID)
 }
+
+// checkHealthContainer check health of container and determine wheather container is ready to accept request or not
+func CheckHealthContainer(socketAddrHttp string) error {
```

Comment:
can we not use the function `BestEffortWaitForHealthy` instead?

Reply:
As per our earlier discussion, We can not use BestEffortWaitForHealthy here because we don't have docker prefix here.